### PR TITLE
enh(install): support for RHEL/CentOS/Oracle Linux v8

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x (master)

### CODE REVIEW

File: en/installation/installation-of-a-poller/using-packages.md [hunk 1]
```diff
@@ -55,17 +70,31 @@ centreon-release package, which will provide the repository file.
 
 Install the Centreon repository using this command:
 
+<!--DOCUSAURUS_CODE_TABS-->
+<!--CentOS 8-->
+```shell
+dnf install -y http://yum.centreon.com/standard/20.10/el8/stable/noarch/RPMS/centreon-release-20.10-2.el7.centos.noarch.rpm
```
Comment (thiuyendang) [inline en/installation/installation-of-a-poller/using-packages.md:76]: el8 instead of el7

**Review by thiuyendang — CHANGES_REQUESTED** [review]

File: en/installation/installation-of-a-poller/using-packages.md (hunk 1 above)
Comment (thiuyendang) [reply en/installation/installation-of-a-poller/using-packages.md:76]: dnf install -y http://yum.centreon.com/standard/20.10/el8/stable/noarch/RPMS/centreon-release-20.10-2.el8.noarch.rpm

File: fr/reporting/installation.md [hunk 2]
```diff
@@ -168,13 +179,23 @@ repository de Centreon MBI.
...
 Lancez la commande suivante :
 
-    yum install centreon-bi-server
+<!--DOCUSAURUS_CODE_TABS-->
+<!--CentOS 8-->
+```shell
+dnf install centreon-bi-server
+```
+<!--CentOS 7-->
+Run the following command:
```
Comment (cgagnaire) [inline fr/reporting/installation.md:188]:
```suggestion
```

File: fr/reporting/installation.md [hunk 3]
```diff
@@ -236,60 +261,80 @@ Ce chapitre aborde l'installation du serveur de reporting.
...
 
-Assurez vous que [le fichier](../assets/reporting/installation/centreon.cnf) de configuration optimisé fourni dans les
-pré-requis est bien présent dans `/etc/my.cnf.d/`, puis redémarrez le service MariaDB
+Assurez vous que [le fichier](../assets/reporting/installation/centreon.cnf) de configuration
+optimisé fourni dans les pré-requis est bien présent dans `/etc/my.cnf.d/`, puis redémarrez
+le service MariaDB :
 
-    systemctl restart mysql
+```shell
+systemctl restart mysql
```
Comment (cgagnaire) [inline fr/reporting/installation.md:301]:
```suggestion
systemctl restart mariadb
```

File: fr/reporting/installation.md [hunk 4]
```diff
@@ -236,60 +261,80 @@ Ce chapitre aborde l'installation du serveur de reporting.
...
 
-    mkdir -p  /etc/systemd/system/mariadb.service.d/
-    echo -ne "[Service]\nLimitNOFILE=32000\n" | tee /etc/systemd/system/mariadb.service.d/limits.conf
-    systemctl daemon-reload
-    systemctl restart mysql
+```shell
+mkdir -p  /etc/systemd/system/mariadb.service.d/
+echo -ne "[Service]\nLimitNOFILE=32000\n" | tee /etc/systemd/system/mariadb.service.d/limits.conf
+systemctl daemon-reload
+systemctl restart mysql
```
Comment (cgagnaire) [inline fr/reporting/installation.md:311]:
```suggestion
systemctl restart mariadb
```

File: fr/reporting/installation.md [hunk 5]
```diff
@@ -236,60 +261,80 @@ Ce chapitre aborde l'installation du serveur de reporting.
...
 
 Si le service MariaDB échoue lors du démarrage, supprimer les fichiers
 *ib_logfile* (MariaDB doit absolument être stoppé) puis redémarrer à
 nouveau MariaDB:
 
-    rm -f /var/lib/mysql/ib_logfile*
-    systemctl start mysql
+```shell
+rm -f /var/lib/mysql/ib_logfile*
+systemctl start mysql
```
Comment (cgagnaire) [inline fr/reporting/installation.md:320]:
```suggestion
systemctl start mariadb
```

File: en/installation/installation-of-a-remote-server/using-packages.md [hunk 6]
```diff
@@ -80,8 +114,14 @@ a remote database on a dedicated server.
 
 <!--With a local database-->
```
Comment (cgagnaire) [inline en/installation/installation-of-a-remote-server/using-packages.md:115]: Tab usage is not the same than Central install

File: en/installation/installation-of-a-central-server/using-packages.md [hunk 7]
```diff
@@ -81,36 +120,56 @@ This section describes how to install a Centreon Central server.
...
 ```
+<!--END_DOCUSAURUS_CODE_TABS-->
 
-<!--With a remote database-->
+### With a remote database
 
 > If installing database on a dedicated server, this server should also have
 > the prerequired repositories.
 
+<!--DOCUSAURUS_CODE_TABS-->
```
Comment (cgagnaire) [inline en/installation/installation-of-a-central-server/using-packages.md:146]: I think we should split this into 2 double tabs section so the tabs are tied to the commands (ie the text "Run the following..." is outside the tabs).
Rendering is better on Docusorus this way.

File: en/reporting/installation.md [hunk 8]
```diff
@@ -172,36 +181,42 @@ central server.
...
-Run the following command: :
-
-    yum install centreon-bi-server
+<!--DOCUSAURUS_CODE_TABS-->
+<!--CentOS 8-->
+```shell
+dnf install centreon-bi-server
+```
+<!--CentOS 7-->
+Run the following command:
```
Comment (cgagnaire) [inline en/reporting/installation.md:190]:
```suggestion
```

File: en/reporting/installation.md [hunk 9]
```diff
@@ -172,36 +181,42 @@ central server.
 Contact the CENTREON service desk to access and install the Centreon MBI
 repository on the Centreon central server.
 
-Run the following command: :
-
-    yum install centreon-bi-server
+<!--DOCUSAURUS_CODE_TABS-->
```
Comment (cgagnaire) [inline en/reporting/installation.md:184]:
```suggestion
Run the following command:

<!--DOCUSAURUS_CODE_TABS-->
```

**Review by cgagnaire — CHANGES_REQUESTED** [review]

File: en/reporting/installation.md [hunk 10]
```diff
@@ -273,95 +301,105 @@ need to add the following GPG key: :
 Make sure you have installed the MariaDB configuration file provided in
 the pre-requisites before starting the MariaDB service [following file](../assets/reporting/installation/centreon.cnf)
 
-    systemctl restart mysql
+```shell
+systemctl restart mysql
```
Comment (cgagnaire) [inline en/reporting/installation.md:305]:
```suggestion
systemctl restart mariadb
```

File: en/reporting/installation.md [hunk 11]
```diff
@@ -273,95 +301,105 @@ need to add the following GPG key: :
...
 
-    mkdir -p  /etc/systemd/system/mariadb.service.d/
-    echo -ne "[Service]\nLimitNOFILE=32000\n" | tee /etc/systemd/system/mariadb.service.d/limits.conf
-    systemctl daemon-reload
-    systemctl restart mysql
+```shell
+mkdir -p  /etc/systemd/system/mariadb.service.d/
+echo -ne "[Service]\nLimitNOFILE=32000\n" | tee /etc/systemd/system/mariadb.service.d/limits.conf
+systemctl daemon-reload
+systemctl restart mysql
```
Comment (cgagnaire) [inline en/reporting/installation.md:315]:
```suggestion
systemctl restart mariadb
```

File: en/reporting/installation.md [hunk 12]
```diff
@@ -273,95 +301,105 @@ need to add the following GPG key: :
...
+```
 
 Then start the service MariaDB. If this service cannot start, remove the
 ib_log files before restarting MariaDB (be sure MariaDB is stopped):
 
-    rm -f /var/lib/mysql/ib_logfile*
-    systemctl restart mysql
+```shell
+rm -f /var/lib/mysql/ib_logfile*
+systemctl restart mysql
```
Comment (cgagnaire) [inline en/reporting/installation.md:323]:
```suggestion
systemctl restart mariadb
```

**Review by cgagnaire — CHANGES_REQUESTED** [review]

**Comment by cgagnaire** [comment]
You should add 4 spaces indentation in the content of the numbered lists where they are used, like:

1. Entering your password:

    ``` shell
   192.168.0.1: please enter your password
   ```

2. Define if you use a proxy to connect to Centreon central or the Remote Server:

   ```shell
   Are you using a proxy ? (y/n)n
   ```

   If you use a proxy, please define credentials:

   ...

**Review by cgagnaire — APPROVED** [review]

**Review by cgagnaire — APPROVED** [review]

**Review by cgagnaire — APPROVED** [review]

**Pull request merged**